code/modules/nano/nanomanager.dm

```diff
@@ -7,6 +7,41 @@
 	var/list/processing_uis = list()
 	// a list of asset filenames which are to be sent to the client on user logon
 	var/list/asset_files = list()
+	// a list of asset files that get added to the RSC directly so as not to lag client joins.
+	var/list/rsc_files = list(
+		'nano/css/codemirror.css',
+		'nano/css/icons.css',
+		'nano/css/layout_basic.css',
+		'nano/css/layout_default.css',
+		'nano/css/lesser-dark.css',
+		'nano/css/shared.css',
```

Comment:
It might actually be a bad idea to make the CSS files part of the rsc-
primarily because it limits debugging ability

With it dynamically sent, you can make changes, restart DD, and test again, quickly
with it in the RSC, you have to recompile, even if you didn't change any DM files
(possibly even clean-compile, I don't fucking want to know how byond handles trash CSS data)

... 
infact i think that's a fairly good point for not doing this at all, is how big of a pain in the ass it makes nano to debug